### PR TITLE
Create dask-tutorial.md

### PR DESCRIPTION
**Pull request by jukent**
Creating a blog post with instructions for @andersy005 's Dask tutorial.

### CODE REVIEW

File: posts/dask-tutorial.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+author: Julia Kent
+date: 2021-07-08
+tags: python-tutorial-series
+---
+
+# Dask Tutorial
+
+The NCAR/UCAR virtual Python Tutorial Seminar Series continues with a 2-part introduction to the Python package `dask` on **Wednesday, July 14th** and **June 23rd at 1 PM Mountain Daylight Time**. These sessions will be led by Anderson Banihirwe.
```
Comment (andersy005) [inline posts/dask-tutorial.md:9]:
```suggestion
The NCAR/UCAR virtual Python Tutorial Seminar Series continues with a 2-part introduction to the Python package `dask` on **Wednesday, July 14th** and **August 11th at 1 PM Mountain Daylight Time**. These sessions will be led by Anderson Banihirwe.
```

File: posts/dask-tutorial.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+If you did not attend the Xarray tutorials:
+
+- Check your conda install if this is the first tutorial in the series you are joining us for with `conda --version` and if necessary [install by following these instructions](https://docs.conda.io/en/latest/miniconda.html).
+
+- Navigate to Anderson's [Xarray Tutorial Github Repository](https://github.com/andersy005/xarray-tutorial).
+
+- Clone this repository by clicking the green "Code" button and copying the SSH address under the word "Clone". This can be done by clicking the clipboard icon.
+
+- In your terminal, type `git clone PASTE` where you paste the SSH address you just copied (`git clone git@github.com:andersy005/xarray-tutorial.git`)
```
Comment (andersy005) [inline posts/dask-tutorial.md:31]: Use the HTTPs address to simplify the instructions?

```suggestion
- Clone this repository by running `git clone https://github.com/andersy005/xarray-tutorial`
```

**Comment by jukent** [comment]
Thanks @andersy005 !

**Review by andersy005 — APPROVED** [review]
This looks great, @jukent! Since @mgrover1 is off this week, I am going to merge this shortly...

**Pull request merged**